internal/data/payments.go

```diff
@@ -150,6 +152,7 @@ func (p *PaymentModel) Get(ctx context.Context, id string, sqlExec db.SQLExecute
 			p.status_history,
 			p.created_at,
 			p.updated_at,
+			p.external_payment_id,
```

Comment:
🛑 all get payment functionality would be broken with this change. You need to either 
`COALESCE(p. external_payment_id, '') as external_payment_id,` or change it to a pointer. 

The reason it fails is fetching a payment that has a nil `external_payment_id` fails. This will be the case for most users. 

<img width="1350" alt="Screenshot 2023-12-22 at 2 06 15 PM" src="https://github.com/stellar/stellar-disbursement-platform-backend/assets/4129193/37206d1a-27be-4600-b985-9c8f151f047a">

Reply:
<img width="1439" alt="Screenshot 2023-12-22 at 1 24 01 PM" src="https://github.com/stellar/stellar-disbursement-platform-backend/assets/20179122/ec67a34f-e954-4261-9ef9-2e79d29290c3">

<img width="1425" alt="Screenshot 2023-12-22 at 1 22 00 PM" src="https://github.com/stellar/stellar-disbursement-platform-backend/assets/20179122/5365ece3-bddc-40c5-b3d7-24d67362db27">